tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -412,6 +412,7 @@ class DuckDBPyRelation:
     def list(self, column: str, groups: str = ..., window_spec: str = ..., projected_columns: str = ...) -> DuckDBPyRelation: ...
 
     def arrow(self, batch_size: int = ...) -> pyarrow.lib.Table: ...
+    def __arrow_c_stream__(self) -> object: ...
```

Comment:
Along with @jorisvandenbossche 's comment, this should be updated to match the spec https://arrow.apache.org/docs/format/CDataInterface/PyCapsuleInterface.html#protocol-typehints

```
    def __arrow_c_stream__(self, requested_schema: object | None = None) -> object: ...
```